### ISSUE TITLE
Diseases from Biomes?

### ISSUE THREAD

**Issue by Omuck3** [Mitchellbrine/DiseaseCraft-2.0#16]
Hi.  Is it possible to make a disease that you can get from a biome?  For example, a jungle biome has a 15% chance of giving you malaria?

**Comment by Mitchellbrine** [comment]
Yes, as far as I remember, that was an original feature of the mod
(although I believe malaria is transferred differently in the current
version, but all that takes is a quick JSON override to change that).

**Comment by Omuck3** [comment]
How would I make a disease come from biome exposure?

**Comment by Mitchellbrine** [comment]
The contracting type is called "biome". It still exists iirc
